tools/protoc-plugin/src/test/java/io/spine/tools/protoc/iface/InterfaceGeneratorTest.java

```diff
@@ -107,176 +108,186 @@ void notAcceptNulls() {
-            Matcher matcher = CUSTOMER_EVENT_INTERFACE_PATTERN.matcher(content);
-            assertTrue(matcher.matches());
+                String content = file.getContent();
+                Matcher matcher = CUSTOMER_EVENT_INTERFACE_PATTERN.matcher(content);
+                assertTrue(matcher.matches());
+            }
         }
-    }
 
-    @Test
-    @DisplayName("generate insertion point contents for Is option")
-    void generateInsertionPointContentsForIsOption() {
-        String filePath = "spine/tools/protoc/iface/is_test.proto";
+        @Test
+        @DisplayName("`Is` option")
+        void generateInsertionPointContentsForIsOption() {
+            String filePath = "spine/tools/protoc/iface/is_test.proto";
 
-        FileDescriptorProto descriptor = IsTestProto.getDescriptor()
-                                                    .toProto();
-        CodeGeneratorResponse response = processCodeGenRequest(filePath, descriptor);
-        assertNotNull(response);
-        List<File> files = response.getFileList();
-        assertEquals(2, files.size());
-        for (File file : files) {
-            assertPackage(file);
-
-            String name = file.getName();
-            String insertionPoint = file.getInsertionPoint();
-            assertFalse(insertionPoint.isEmpty());
-            String content = file.getContent();
-            if (name.endsWith("ProtocNameUpdated.java")) {
-                assertTrue(content.contains("Event,"));
-            } else if (name.endsWith("ProtocUpdateName.java")) {
-                assertTrue(content.contains("Command,"));
+            FileDescriptorProto descriptor = IsTestProto.getDescriptor()
+                                                        .toProto();
+            CodeGeneratorResponse response = processCodeGenRequest(filePath, descriptor);
+            assertNotNull(response);
+            List<File> files = response.getFileList();
+            assertEquals(2, files.size());
+            for (File file : files) {
+                assertPackage(file);
+
+                String name = file.getName();
+                String insertionPoint = file.getInsertionPoint();
+                assertFalse(insertionPoint.isEmpty());
+                String content = file.getContent();
+                if (name.endsWith("ProtocNameUpdated.java")) {
+                    assertTrue(content.contains("Event,"));
+                } else if (name.endsWith("ProtocUpdateName.java")) {
+                    assertTrue(content.contains("Command,"));
+                }
             }
         }
-    }
 
-    @Test
-    @DisplayName("generate insertion point contents for EveryIs in singe file")
-    void generateInsertionPointContentsForEveryIsInSingleFile() {
-        String filePath = "spine/tools/protoc/iface/every_is_in_one_file.proto";
+        @Test
+        @DisplayName("`EveryIs` in singe file")
+        void generateInsertionPointContentsForEveryIsInSingleFile() {
```

Comment:
How is this test different from the next one called `void isInSingleFile`?

Reply:
This one is on `EveryIs`, while the next one is on `Is`.